mobile/src/screens/meditation-list/components/meditation-item/meditation-item.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { type ImageSourcePropType } from 'react-native';
+
+import ImagePlaceholder from '#assets/img/card-image-placeholder.png';
+import Play from '#assets/img/icons/play.svg';
```

Comment:
```suggestion
import cardImagePlaceholder from '#assets/img/card-image-placeholder.png';
import play from '#assets/img/icons/play.svg';
```

Reply:
I can't use 'play' in lowercase because I receive this error: Property 'play' does not exist on type 'JSX.IntrinsicElements'.